src/pages/dashboard/ui/ticket/TicketOptionPage.tsx

```
@@ -1,229 +1,69 @@
+    queries: tickets.map(ticket => ({
+      queryKey: ['attachedTicketOptions', ticket.ticketId],
+      queryFn: () => getAttachedTicketOptions(ticket.ticketId),
+      enabled: !!ticket.ticketId,
+    })),
+  });
+
+  // 티켓별 옵션 맵핑
+  const attachedOptionsMap = tickets.reduce((acc, ticket, idx) => {
+    const query = attachedOptionsQueries[idx];
+    if (query && query.data && query.data.result) {
+      acc[ticket.ticketId] = Array.isArray(query.data.result) ? query.data.result : [query.data.result];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)